pkg/validate/accounts.go

```diff
@@ -38,16 +38,6 @@ func getAllAccounts(spinSvc v1alpha2.SpinnakerServiceInterface, accountType acco
 			return nil, err
 		}
 	}
-	// Get CRD accounts if enabled
```

Comment:
Where was this logic moved to validate CRD accounts?

Reply:
CRD accounts should be validated by the CRD validator not here.